mods/cnc/maps/gdi06/map.yaml

```diff
@@ -0,0 +1,1166 @@
+	MiniGunner1: e1
+		Location: 47,13
+		Owner: Nod
+		SubCell: 4
+	MiniGunner2: e1
+		Location: 47,14
+		Owner: Nod
+		SubCell: 0
+	Actor264: e3
+		Location: 52,17
+		Owner: Nod
+		Facing: 160
+		SubCell: 1
+	Actor265: e3
+		Location: 52,17
+		Owner: Nod
+		Facing: 160
+		SubCell: 2
+	Actor266: e3
+		Location: 47,17
+		Owner: Nod
+		Facing: 96
+		SubCell: 1
+	Actor267: e3
+		Location: 46,17
+		Owner: Nod
+		Facing: 96
+		SubCell: 2
+	AttackTrigger2: e1
+		Location: 39,6
+		Owner: Nod
+		Facing: 128
+		SubCell: 0
+	Actor269: e3.sticky
+		Location: 40,25
+		Owner: Nod
+		Facing: 96
+		SubCell: 0
+	Actor270: e1
+		Location: 53,42
+		Owner: Nod
+		SubCell: 0
+	Actor271: e3
+		Location: 43,40
+		Owner: Nod
+		Facing: 96
+		SubCell: 4
+	Actor272: e1
+		Location: 49,43
+		Owner: Nod
+		Facing: 96
+		SubCell: 4
+	Actor273: e1
+		Location: 50,43
+		Owner: Nod
+		Facing: 128
+		SubCell: 1
+	Actor274: e1
+		Location: 53,47
+		Owner: Nod
+		Facing: 32
+		SubCell: 3
+	CivFleeTrigger: e1
+		Location: 23,36
+		Owner: Nod
+		Facing: 224
+		SubCell: 3
+	Civilian: c8
+		Location: 22,36
+		Owner: Neutral
+		Facing: 96
+		SubCell: 3
+	CivHideOut: waypoint
+		Location: 26,32
+		Owner: Neutral
+	MiniGunner5: e1
+		Location: 51,20
+		Owner: Nod
+		SubCell: 1
+	MiniGunner6: e1
+		Location: 51,20
+		Owner: Nod
+		SubCell: 2
+	MiniGunner3: e1
+		Location: 48,20
+		Owner: Nod
+		SubCell: 1
+	MiniGunner4: e1
+		Location: 48,20
+		Owner: Nod
+		SubCell: 2
+	waypoint27: waypoint
+		Location: 32,32
+		Owner: Neutral
+	Flare: waypoint
+		Location: 30,41
+		Owner: Neutral
+	waypoint17: waypoint
+		Location: 47,22
+		Owner: Neutral
+	waypoint16: waypoint
+		Location: 52,22
+		Owner: Neutral
+	waypoint15: waypoint
+		Location: 54,22
+		Owner: Neutral
+	waypoint14: waypoint
+		Location: 45,22
+		Owner: Neutral
+	waypoint13: waypoint
+		Location: 18,18
+		Owner: Neutral
+	waypoint12: waypoint
+		Location: 20,7
+		Owner: Neutral
+	waypoint11: waypoint
+		Location: 60,10
+		Owner: Neutral
+	waypoint10: waypoint
+		Location: 60,23
+		Owner: Neutral
+	waypoint9: waypoint
+		Location: 38,19
+		Owner: Neutral
+	waypoint8: waypoint
+		Location: 24,18
+		Owner: Neutral
+	waypoint7: waypoint
+		Location: 36,32
+		Owner: Neutral
+	waypoint6: waypoint
+		Location: 49,24
+		Owner: Neutral
+	waypoint5: waypoint
+		Location: 39,13
+		Owner: Neutral
+	waypoint4: waypoint
+		Location: 39,7
+		Owner: Neutral
+	waypoint3: waypoint
+		Location: 26,27
+		Owner: Neutral
+	waypoint2: waypoint
+		Location: 27,32
+		Owner: Neutral
+	TransportRally: waypoint
+		Location: 36,42
+		Owner: Neutral
+	UnitsRally: waypoint
+		Owner: GDI
+		Location: 55,52
+	lstEnd: waypoint
+		Owner: GDI
+		Location: 55,53
+	lstStart: waypoint
+		Owner: GDI
+		Location: 55,61
+
+Smudges:
+
+Rules:
+	World:
+		-SpawnMPUnits:
+		-MPStartLocations:
+		-CrateSpawner:
+		LuaScript:
+			Scripts: gdi06.lua
+		ObjectivesPanel:
+			PanelName: MISSION_OBJECTIVES
+		MusicPlaylist:
+			BackgroundMusic: ambientrain
+			StartingMusic: rain
```

Comment:
I added BackgroundMusic after Mailänder's comment (see above); the trait documentation says nothing about the *Music traits being mutually exclusive.
The idea is to play the Rain in the Night track but have the ambient music in case the track is stopped - does it work for you?